Markdown examples update (#656)
Codecademy/docs: content/markdown/concepts/emphasis/emphasis.md

**bold text**
```

For example:

> ```markdown

Reviewer: Could we change all instances of \```markdown to \```md, per the [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md#code-blocks)?

Author: Totally! Done in [b03c375](https://github.com/Codecademy/docs/pull/656/commits/b03c375e2e651107a2a9fd390c12e4c6a12f7efb)

Please do **NOT** touch your pimples.
```

> Will render as:

Reviewer: ```suggestion
The output would be:
```

_italic text_
```

For example:

```markdown
This avacado egg sandwich is _very_ good.
```

The output would be:

> This avacado egg sandwich is _very_ good.

Reviewer 2: ```suggestion
This avocado egg sandwich is _very_ good.
```

🥑

Reviewer 2: @mikikiv This is the only other thing noticed 😆

## Strikethrough

> To make a strikethrough text, use 2 tildes `~` before and after a piece of text.

Reviewer: ```suggestion
A strikethrough can be applied to text by surrounding with two tildes `~~`, like in the example below:
```

And then I would consolidate the following two snippets into one that looks like this:

```md
I did ~~not~~ build a website.
```
